isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,355 @@
+#!/bin/bash
+
+# Define environment name and package to install
+ISIS_LABEL="main"
+DOWNLOAD_DATA="YES"
+
+# Report error and reserves error code from a failed command
+failed_command() {
+    local _status=$?
+    if [ "${_status}" -ne 0 ]; then 
+        echo "Error: $1 failed" 
+        exit ${_status}
+    fi
+}
+
+print_help() {
+    printf "Usage: $0 [options] ISIS_LABEL ISIS_VERSION\n"
+    printf "Options:\n"
+    printf "\t-h, --help            Show this help message and exit\n"
+    printf "\t-n, --env_name        The name of the anaconda environment to create.\n"
+    printf "\t-m, --miniforge_dir   Define the directory to an anaconda package manager install location. "
+    printf "If you have an anaconda package manager already this argument will be ignored. If not "
+    printf "a version of miniforge will be installed at this location\n"
+    printf "\t-p, --data_prefix     The directory where ISISDATA is located. If this "
+    printf "directory doesn't exist then one will be made at its location\n"
+    printf "\t--no_data             Do not ask to download any data for the ISIS_DATA area.\n"
+    printf "\n"
+    printf "\tDefining variables on the command line will skip the interactive "
+    printf "elements within this script"
+    printf "\n\n"
+    printf "\t\$1 ISIS_LABEL - Different ISIS labels as defined by the "
+    printf "anaconda labels at https://anaconda.org/usgs-astrogeology/isis"
+    printf ", examples include \"LTS\", \"dev\", and \"RC\"\n"
+    printf "\t\$2 ISIS_VERSION - Different ISIS versions as defined by the "
+    printf "anaconda versions for a label at https://anaconda.org/usgs-astrogeology/isis"
+    printf ", examples include 8.0.3 (LTS/Feature), 8.2.0_RC1 (RC), and 2025.02.22 (dev)"
+    exit 1
```

Comment:
Consider omitting the `exit` from the `print_help()` function so that exit can be called immediately after invoking the function but using an exit code appropriate for the context. For example, if the user invokes the script with the `-h` option, the script should print the help message and exit normally (code 0) because there is no error.